recipes/ccache/all/conanfile.py

```diff
@@ -79,35 +88,43 @@ def package_id(self):
         del self.info.settings.compiler
 
     def build_requirements(self):
-        if hasattr(self, "settings_build") and tools.cross_building(self) and \
+        self.build_requires("pkgconf/1.9.3")
+        if hasattr(self, "settings_build") and cross_building(self) and \
            self.settings.os == "Macos" and self.settings.arch == "armv8":
-            self.build_requires("cmake/3.22.0")
+            self.tool_requires("cmake/3.25.1")
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
-                  destination=self._source_subfolder, strip_root=True)
+        get(self, **self.conan_data["sources"][self.version],
+                  destination=self.source_folder, strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["REDIS_STORAGE_BACKEND"] = self.options.redis_storage_backend
+        tc.variables["HIREDIS_FROM_INTERNET"] = False
+        tc.variables["ENABLE_DOCUMENTATION"] = False
+        tc.variables["ENABLE_TESTING"] = False
+        tc.generate()
+        deps = PkgConfigDeps(self)
+        deps.generate()
+        if is_msvc(self):
+            deps = CMakeDeps(self)
+            deps.set_property("hiredislib", "cmake_target_name", "HIREDIS::HIREDIS")
+            deps.set_property("zstd", "cmake_target_name", "ZSTD::ZSTD")
+            deps.generate()
```

Comment:
```suggestion
        deps = CMakeDeps(self)
        deps.set_property("hiredislib", "cmake_target_name", "HIREDIS::HIREDIS")
        deps.set_property("zstd", "cmake_target_name", "ZSTD::ZSTD")
        deps.generate()
```
Use CMakeDeps only, it should be enough for any platform.

Reply:
Unfortunately that doesn't work, as per CI log.

Reply:
I'll take a look locally this afternoon. Tahnk you for pinging

Reply:
Thanks :)